misk/src/main/kotlin/misk/eventrouter/RealEventRouterModule.kt

```diff
@@ -18,10 +19,15 @@ import javax.inject.Singleton
 
 class RealEventRouterModule : KAbstractModule() {
   override fun configure() {
+    val environment = Environment.fromEnvironmentVariable()
```

Comment:
This should be `@Inject lateinit var environment: Environment`

Reply:
ahh, you can’t ’cause this is itself a module. Pass the environment to the module constructor?